squad/ci/backend/lava.py

```diff
@@ -260,11 +260,13 @@ def __parse_results__(self, data, test_job):
                                 any(substring.lower() in metadata['error_msg'].lower() for substring in LAVA_INFRA_ERROR_MESSAGES):
                             if test_job.resubmitted_count < 3:
                                 resubmitted_job = self.resubmit(test_job)
-                                send_testjob_resubmit_admin_email.delay(test_job.pk, resubmitted_job.pk)
+                                # delay sending email by 15 seconds to allow the database object to be saved
+                                send_testjob_resubmit_admin_email.apply_async(args=[test_job.pk, resubmitted_job.pk], countdown=15)
```

Comment:
What happens without the delay?

Reply:
Notification is sent before the testjob.status is saved, so it's reported as 'None'. Not a big deal, but I guess it's better to wait a moment.